portality/tasks/harvester_helpers/epmc/client.py

```diff
@@ -58,6 +58,7 @@ def __init__(self, message, rawstring, *args, **kwargs):
 
 
 class EuropePMC(object):
+    job = None
```

Comment:
The EMPC client should not know about our job queue.  For example, what if we want to run the client independently of the job queue, or integrate it into some other part of the system as well?  Architecturally this is not a good approach, it creates a two-way binding between modules which should only have a one-way binding.

The client should be a generic client, and the job queue should wrangle it.  Looking into why the job is passed in, it's for the recording of logs, and for no other functional purpose.  If that's the case I'd suggest 2 alternative approaches:

1. pass in a "logger" for the client to use.  This could be provided by the job queue to route the logs to the job's audit trail
2. Have the client log to a state variable (e.g. EuropePMC.log) and have the task read the logs into its own audit logs after the event.